src/kv/untyped_tx_view.h

```diff
@@ -112,6 +112,52 @@ namespace kv::untyped
       return found.value;
     }
 
+    /** Test if key is present
+     *
+     * This returns true if the key has a value inside the transaction. If the
+     * key has been updated (written to or deleted) in the current transaction,
+     * that update will be reflected in the return of this call.
+     *
+     * @param key Key
+     *
+     * @return bool true iff key exists
+     */
+    bool has(const KeyType& key)
+    {
+      // NB: In terms of dependencies, this is a read equivalent to a get
```

Comment:
Why not calling `get(key).has_value()` here?

Reply:
Because `get(key)` constructs a copy of the value, which we'd rather not do if we don't have to. I'll try to rewrite this so they both call the same access non-copying function to avoid this duplication.